internal/cmd/skupper/listener/kube/listener_create.go

```diff
@@ -0,0 +1,217 @@
+package kube
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/skupperproject/skupper/client"
+	"github.com/skupperproject/skupper/internal/cmd/skupper/utils"
+	"github.com/skupperproject/skupper/pkg/apis/skupper/v1alpha1"
+	skupperv1alpha1 "github.com/skupperproject/skupper/pkg/generated/client/clientset/versioned/typed/skupper/v1alpha1"
+	"github.com/skupperproject/skupper/pkg/utils/validator"
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var (
+	listenerCreateLong    = "Clients at this site use the listener host and port to establish connections to the remote service."
+	listenerCreateExample = "skupper listener create database 5432"
+)
+
+type ListenerCreate struct {
+	routingKey   string
+	host         string
+	tlsSecret    string
+	listenerType string
+	output       string
+}
+
+type CmdListenerCreate struct {
+	client     skupperv1alpha1.SkupperV1alpha1Interface
+	CobraCmd   cobra.Command
+	flags      ListenerCreate
+	namespace  string
+	name       string
+	port       int
+	output     string
+	KubeClient kubernetes.Interface
+}
+
+func NewCmdListenerCreate() *CmdListenerCreate {
+
+	skupperCmd := CmdListenerCreate{flags: ListenerCreate{}}
+
+	cmd := cobra.Command{
+		Use:     "create <name>",
```

Comment:
"create <name> <port>"

Reply:
fixed

Reply:
I mean the `Use` value should be "create <name> <port>"